src/main/java/org/cactoos/collection/Synced.java

```diff
@@ -69,12 +69,14 @@ public Synced(final Iterable<T> src) {
      */
     public Synced(final Collection<T> src) {
         super(
-            new org.cactoos.scalar.Synced<>(
-                () -> {
-                    final Collection<T> temp = new LinkedList<>();
-                    temp.addAll(src);
-                    return Collections.synchronizedCollection(temp);
-                }
+            new CollectionOf<>(
+                new org.cactoos.scalar.Synced<>(
+                    () -> {
+                        final Collection<T> temp = new LinkedList<>();
+                        temp.addAll(src);
```

Comment:
@victornoel please, change `temp.addAll(src)` to `src.forEach(temp::add)` to be more homegeneus.